packages/react-conformance/src/conformance.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Details } from './types';
+
+function isConformant(Component: React.ComponentType, options: Details = {}) {
```

Comment:
Instead of export default below I'd prefer that you just do `export function isConformant` and remove the default. (this is a stylistic difference between v0 and v7 which we haven't quite resolved.)